app/livechat/server/hooks/sendToFacebook.js

```diff
@@ -16,6 +17,7 @@ callbacks.add('afterSaveMessage', function(message, room) {
 	if (!(typeof room.t !== 'undefined' && room.t === 'l' && room.facebook && room.v && room.v.token)) {
 		return message;
 	}
+	FileUpload.addJWTToFileUrlIfNecessary(message);
```

Comment:
As I commented before, IMO it would be better having a specific helper method to handle this process, and inside that method, I would call the addJWTToFileUrlIfNecessary method.
In that case, we could rename the addJWTToFileUrlIfNecessary to addJWTToFileUrl.